modules/apps/29-fee/ibc_module.go

```diff
@@ -224,9 +222,10 @@ func (im IBCModule) OnAcknowledgementPacket(
 	// cache context before trying to distribute the fee
 	cacheCtx, writeFn := ctx.CacheContext()
 
-	fmt.Println("forward", ack.ForwardRelayerAddress)
+	forwardRelayer, _ := sdk.AccAddressFromBech32(ack.ForwardRelayerAddress)
```

Comment:
we should probably check the error

Reply:
yes...but then should we forego distribution entirely if this error exists?

Reply:
I think we handle this here already implicitly -> https://github.com/cosmos/ibc-go/blob/feat/ics-29-packetcallbacks/modules/apps/29-fee/keeper/escrow.go#L56 

So no need to return an error here